Oglle/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"author": {
+		"name": "Oglle",
+		"id": "641614088215986186"
+	},
+	"service": "OGLLE.COM.BR",

```

Comment:
Service should be "Oglle"
Author name and id should be YOUR discord name and id.